### DISCUSSION TITLE
TIM HUB DGA4132 missing IP address

### DISCUSSION THREAD

**Discussion by SmooKeeIce** [hack-technicolor/hack-technicolor#309] [General]
Hi all guys, I've encountered a problem with my old TIM modem router.
I was going to install Ansuel GUI in order to turn it into a Wi-Fi repeater (I already have a main router for my home network), so I followed all the steps until I downgraded the firmware to version 1.0.3, once installed the modem completely lost its IP address, I also tried scanning the addresses from the command prompt but the default gateway comes up empty whether I connect via LAN or Wi-Fi.
I have also tried doing a hard reset but the situation remains the same.
The modem seems to be working properly (the "WI-Fi 5" and "Wi-Fi 2.5" lights up green and "INFOLED" lights up red) but I can no longer access it because the IP has disappeared.
I don't know what to do anymore I hope you can help me

**Comment by FrancYescO** [comment]
reset again.

**Comment by SmooKeeIce** [comment]
> reset again.

Already done several times, nothing changes, the IP address does not appear

**Comment by ghost** [comment]
Hello @SmooKeeIce,

Have you considered assigning a static IP address to the interface? Are you able to receive any traffic? Can you observe any data when utilizing tcpdump for network packet analysis?

What  is ifconfig -a show? Do you have interface up and running? 

There is really much things that is unknown here so it's hard to answer this question. I assume you have shell access?
